### DISCUSSION TITLE
Video

### DISCUSSION THREAD

**Discussion by sabanadesan** [StereoKit/StereoKit#625] [Q&A]
Can you show some example code on projecting video onto texture or object?

**Comment by maluoi** [comment]
I'm afraid I also don't have an example of this on-hand either! Video playback is doable with StereoKit, but you will need an external library for decoding video. At one point I did have a partially working demo of LibVLCSharp working in SK, but I hadn't nailed it down completely (also on a different computer I won't have access to until next week 😅 ). I've also heard someone mention that OpenCV was pretty easy to us for video playback?

I do however have some samples that involve displaying color buffers from external tools! [The Browser Sample](https://github.com/StereoKit/StereoKit-Browser) in particular displays a browser surface in SK that you can interact with, which may provide some decent reference for interfacing with a video playback package :)

**Comment by salexx** [comment]
Hello. Could you please show the example code for the texture video. Even unfinished code would greatly simplify the task. Thank you!!!

**Reply by nioroso-x3** [reply]
In the main.cpp of SK-FPV you can see how to display video using opencv. Basically you take the RGBA decoded images and write them to a texture.

https://github.com/nioroso-x3/SK-FPV/blob/main/src/main.cpp
